lib/selectivity/rspec.rb

```diff
@@ -44,25 +44,25 @@ def find_selectivity_input(from, options)
         find(:div, "##{label[:for]}", options)
       end
 
-      def multiselect?(input)
+      def _selectivity_multiselect?(input)
         input.first('.selectivity-multiple-input-container').present?
       end
 
-      def select!(input, item)
+      def _selectivity_select!(input, item)
         input.click
 
         within 'div.selectivity-dropdown' do
-          if (find(:xpath, "//div[@class='selectivity-result-item'][contains(text(), '#{item}')]")).visible?
+          if find('div.selectivity-result-item', text: item).visible?
```

Comment:
This is an improvement only on a first sight. Once you have a lot of options in the select (e.g. https://github.com/msx2/selectivity-rails-sample-app), it takes *a lot* more time to browse through all the options and find a specific one than using `xpath` ;)